outsettingsvalues/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+outsettingsvalues app settings file.
```

Comment:
Seems it docstring relates to some particular file. Could you update it to reflect package content or in case, when we actually do not need this docstring - add some notation to ignore missing docstring?